code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -386,7 +386,24 @@ meteor_act
 					adjustOxyLoss(10)
 					adjustHalLoss(5)
 
-
+				if("winded")
+					visible_message(SPAN_WARNING("[src] is winded!"), SPAN_DANGER("You feel disoriented!"))
+					confused = max(confused, 2)
+					external_recoil(40)
+					var/obj/item/item_in_active_hand = get_active_hand()
+					var/mob/living/carbon/M = user
+					if(recoil >= 60 && item_in_active_hand)
+						if(istype(item_in_active_hand, /obj/item/grab))
+							break_all_grabs(M) //See about breaking grips or pulls
+							playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+							return TRUE
+						if(item_in_active_hand.wielded && recoil < 80)
+							playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
+							return TRUE
+						unEquip(item_in_active_hand)
+						visible_message(SPAN_DANGER("[M] has disarmed [src]!"))
```

Comment:
```suggestion
					if(recoil >= 60 && item_in_active_hand)
						if(istype(item_in_active_hand, /obj/item/grab))
							break_all_grabs(user) //See about breaking grips or pulls
							playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
							return TRUE
						if(item_in_active_hand.wielded && recoil < 80)
							playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
							return TRUE
						unEquip(item_in_active_hand)
						visible_message(SPAN_DANGER("[user] has disarmed [src]!"))
```
No need to cast as `lmob/living/carbon`, `break_all_grab` will automatically do that since it is `break_all_grabs(mob/living/carbon/user)`